web/components/ImportButton.vue

```diff
@@ -103,12 +108,13 @@ export default defineComponent({
           data.dialog = false;
         });
     }
-    function readfile(e) {
+    function readfile(e: { target: { files: any[] } }) {
       const file = e.target.files[0];
       const reader = new FileReader();
 
       reader.onload = function () {
-        data.filedata = reader.result.toString();
+        data.filedata = reader.result as string;
```

Comment:
If possible, we should check the file's format is suit for. (But I have no good idea, yet)

Reply:
You can use `yaml.load` to parse yaml like:
https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/7ff2258b58f38e6a64a9d931806e9f5220623d01/web/components/lib/template.ts#L12

And you should define a class which indicate the exported file format and can check whether the file is correct or not like:

```
    const filedate:YouDefinedClass = yaml.load(reader.result);
```

Reply:
oh... It looks like a lot of work to define the classes...ok👌

Reply:
I fixed it👌